src/shared/utils/challenge-listing/filter.js

```diff
@@ -73,6 +73,9 @@ function filterByGroupIds(challenge, state) {
 function filterByRegistrationOpen(challenge, state) {
   if (_.isUndefined(state.registrationOpen)) return true;
   const isRegOpen = () => {
+    if ('registrationOpen' in challenge) {
```

Comment:
No, please do such checks just as `if (challenge.registrationOpen) {`.
I am also not sure, why do we need to touch this filter at all?

Reply:
@birdofpreyru This filter needed to be changed because the "open registration" challenge listing included results for stalled challenges which were closed a long time ago, but still appeared because they had a "status' key set to ACTIVE. There is another key in all challenge results that was not being checked in the filter called "registrationOpen", and that key was properly set to "No" on these stalled challenges, so doing this extra check here removes these old stalled, closed challenges from the open registration listings.